### PR TITLE
Extended information about additional registration of APIML

### PR DESCRIPTION
**Pull request by pj892031**
The PR tries to clarify what systems are defined in the `components.gateway.apiml.service.additionalRegistration.*.discoveryServiceUrls`. The aim is to explain difference between APIML instance and DS instance and how to configure sysplex and monoplex.

The PR was based on https://github.com/zowe/api-layer/issues/3823

There are also fixes of YAML structure. The indentation and `-` characters are important.

List the file(s) included in this PR:

After creating the PR, follow the instructions in the comments.

### CODE REVIEW

File: docs/user-guide/api-mediation/api-mediation-multi-tenancy.md [hunk 1]
```diff
@@ -67,12 +67,21 @@ Use the following example as a template for how to set the value for this proper
...
-       - discoveryServiceUrls:      https://sys1:{discoveryServicePort}/eureka/,https://sys2:{discoveryServicePort}/eureka/
- 	    routes:
+       - discoveryServiceUrls: https://sys1:{discoveryServicePort}/eureka/,https://sys2:{discoveryServicePort}/eureka/
+ 	     routes:
               - gatewayUrl: /
                 serviceUrl: /
 ```
 
+:::note 
+  The value `discoveryServiceUrls` should contain all instances of Discovery services for a specific API ML instance.
```
Comment (balhar-jakub) [inline docs/user-guide/api-mediation/api-mediation-multi-tenancy.md:77]: The specific API ML instance needs bit more explanation. I was confused when reading this.

File: docs/user-guide/api-mediation/api-mediation-multi-tenancy.md [hunk 2]
```diff
@@ -119,12 +128,21 @@ Use the following example as a template for how to set the value of this propert
...
     # central API ML (in HA, for non-HA mode use only 1 hostname)
-       - discoveryServiceUrls:      https://sys1:{discoveryServicePort}/eureka/,https://sys2:{discoveryServicePort}/eureka/
- 	    routes:
+       - discoveryServiceUrls: https://sys1:{discoveryServicePort}/eureka/,https://sys2:{discoveryServicePort}/eureka/
+ 	     routes:
               - gatewayUrl: /
                 serviceUrl: /
 ```
 
+:::note
```
Comment (janan07) [inline docs/user-guide/api-mediation/api-mediation-multi-tenancy.md:137]: See previous comment.

File: docs/user-guide/api-mediation/api-mediation-multi-tenancy.md [hunk 3]
```diff
@@ -137,6 +155,14 @@ ZWE_CONFIGS_APIML_SERVICE_ADDITIONALREGISTRATION_0_ROUTES_0_GATEWAYURL=/
 ZWE_CONFIGS_APIML_SERVICE_ADDITIONALREGISTRATION_0_ROUTES_0_SERVICEURL=/
 ```
 
+:::note
+  Value `#` in the properties name `ZWE_CONFIGS_APIML_SERVICE_ADDITIONALREGISTRATION_#_*` defines ID of APIML instance.
```
Comment (janan07) [inline docs/user-guide/api-mediation/api-mediation-multi-tenancy.md:159]: See previous comment.

File: docs/user-guide/api-mediation/configuration-multi-tenancy-routing.md [hunk 4]
```diff
@@ -13,18 +13,26 @@ Follow these steps to register with additional Discovery Services:
...
-                    routes:
-                            gatewayUrl: /
-                            serviceUrl: /
+         routes:
+              - gatewayUrl: /
+                serviceUrl: /
     ```
+
+   :::note
+     The value `discoveryServiceUrls` should contain all instances of Discovery services for a specific API ML instance.
```
Comment (janan07) [inline docs/user-guide/api-mediation/configuration-multi-tenancy-routing.md:32]: Previous comment

File: docs/user-guide/api-mediation/api-mediation-multi-tenancy.md (hunk 1 above)
Comment (balhar-jakub) [reply docs/user-guide/api-mediation/api-mediation-multi-tenancy.md]: I believe that the issue here is that there is concept of multiple domains. Where each of the domains have Zowe (Expectation is that Zowe will be setup in HA and therefore have more instances of every service)

On top of that there is a Gateway ahead of all these domains handling routing and management of all the domains. 

We need to make sure that we properly distinguish between these two pieces of software.

**Review by janan07 — APPROVED** [review]
reviewed and edited by doc squad

**Pull request merged**